onnxruntime/contrib_ops/cpu/skip_layer_norm.cc

```diff
@@ -118,12 +183,15 @@ Status SkipLayerNorm<T, simplified>::Compute(OpKernelContext* p_ctx) const {
         }
 
         for (int64_t h = 0; h < hidden_size; h++) {
+          DoubleOrFloat output_value = ConvertMLFloat16ToDoubleOrFloatIfNeeded<T, DoubleOrFloat>(p_output[h]);
```

Comment:
I wonder if there is a way to compare what pytorch does. output_value is computed then converted to float16, then converted to float before being normlized. I wonder if pytorch is avoided that rounding to float16.

Reply:
updated in the latest iteration